rtCommon/bidsRun.py

```diff
@@ -166,6 +167,15 @@ def appendIncremental(self, incremental: BidsIncremental,
         # multiple images in its image volume
         imagesInVolume = incremental.imageDimensions[3]
 
+        try:
+            import indexed_gzip
+        except ImportError:
+            warnings.warn("Package indexed_gzip not available. If you're "
+                          "trying to append a BIDS Incremental that uses a "
+                          "gzipped NIfTI file as its underlying data source, "
+                          "appendIncremental will run very slowly! We strongly "
+                          "recommend you install the indexed_gzip package.")
```

Comment:
Maybe add to the warning here the command to install, e.g. 'conda install indexed_gzip'